docker/inference/dockerfile.ctr

```diff
@@ -1,5 +1,5 @@
 # syntax=docker/dockerfile:1.2
-ARG TRITON_VERSION=22.01
+ARG TRITON_VERSION=22.02
```

Comment:
Do we want to make this a range? >= ?

Reply:
This is a variable, it it not a comparison, it is an assignation